README.rst

```diff
@@ -91,6 +91,28 @@ that you have the ``universe`` repository enabled. The ``universe`` repository
 is enabled by default in a standard Ubuntu installation but may not be on some
 minimal, or not standard, installations.
 
+On Docker::
+
+Required dependencies installed on the target system:
+  - Docker
+  - docker-compose
+
+You can pull the image from our Docker Hub
+
+        docker pull openobservatory/ooni-probe:latest
+        docker run -p 8842:8842 -d openobservatory/ooni-probe:latest
+
+If you prefer to run the ooni-probe in a more persistent manner on your host
+you can add `--restart always` to the execution.
+
+        docker run -p 8842:8842 --restart always -d openobservatory/ooni-probe:latest
+
+On the other hand you can also build the image on your local machine:
+
+        git clone https://github.com/TheTorProject/ooni-probe.git
+        docker-compose build
+        docker-compose up
```

Comment:
Can you replace these two commands with just vanilla docker (without docker-compose) and remove the docker-compose dependency in the above lines?